CameraTutorial/Game.cs

```diff
@@ -100,7 +102,7 @@ protected override void OnLoad(EventArgs e)
             Camera.Init(45, Width / Height);
```

Comment:
There should be some mechanism for changing the aspect ratio on the camera, since the window can be resized, invalidating this.